### PR TITLE
Fix post preview with location

### PR DESCRIPTION
**Pull request by svbergerem**

### CODE REVIEW

File: app/assets/javascripts/app/views/publisher_view.js [hunk 1]
```diff
@@ -327,6 +327,17 @@ app.views.Publisher = Backbone.View.extend({
     var mentionedPeople = this.getMentionedPeople(serializedForm);
     var date = (new Date()).toISOString();
     var poll = this.getPollData(serializedForm);
+    var location_coords = serializedForm["location[coords]"];
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/publisher_view.js]: Identifier 'location_coords' is not in camel case.

File: app/assets/javascripts/app/views/publisher_view.js [hunk 2]
```diff
@@ -327,6 +327,17 @@ app.views.Publisher = Backbone.View.extend({
...
+    if(location_coords === "") {
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/publisher_view.js]: Identifier 'location_coords' is not in camel case.

File: app/assets/javascripts/app/views/publisher_view.js [hunk 3]
```diff
@@ -327,6 +327,17 @@ app.views.Publisher = Backbone.View.extend({
...
+      location_coords = ["", ""];
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/publisher_view.js]: Identifier 'location_coords' is not in camel case.

File: app/assets/javascripts/app/views/publisher_view.js [hunk 4]
```diff
@@ -327,6 +327,17 @@ app.views.Publisher = Backbone.View.extend({
...
+    } else {
+      location_coords = location_coords.split(",");
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/publisher_view.js]: Identifier 'location_coords' is not in camel case.

File: app/assets/javascripts/app/views/publisher_view.js [hunk 5]
```diff
@@ -327,6 +327,17 @@ app.views.Publisher = Backbone.View.extend({
...
+    }
+    var location = {
+      "address": $("#location_address").val(),
+      "lat": location_coords[0],
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/publisher_view.js]: Identifier 'location_coords' is not in camel case.

File: app/assets/javascripts/app/views/publisher_view.js [hunk 6]
```diff
@@ -327,6 +327,17 @@ app.views.Publisher = Backbone.View.extend({
...
+      "lng": location_coords[1]
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/publisher_view.js]: Identifier 'location_coords' is not in camel case.

**Comment by jhass** [comment]
No spec? :(

**Comment by svbergerem** [comment]
> No spec? :(

Done.

**Comment by jhass** [comment]
Thank you!

**Pull request merged**